README.md

```diff
@@ -494,4 +494,6 @@ CUnit CPPUnit Checker
 For configuration please have a look into README_feature_CUnitCppUnit_Checker.txt.
 
 
+[Bug tracker, KIT]: https://github.com/KITPraktomatTeam/Praktomat/issues
+[Bug tracker, HSO]: https://github.com/hso-praktomat/praktomat/issues
```

Comment:
Here is the KIT repository reference ...
https://github.com/KITPraktomatTeam/Praktomat/blame/master/README.md#L376

Reply:
That's not what I meant :D

Markdown doesn't render these links unless we're explicitly referring somewhere to them.
The following would produce no output:
```markdown
[Bug tracker, KIT]: https://github.com/KITPraktomatTeam/Praktomat/issues
```

However, this would produce a link titled "Bug tracker, KIT" referring to their bug tracker:
```markdown
[Bug tracker, KIT]

[Bug tracker, KIT]: https://github.com/KITPraktomatTeam/Praktomat/issues
```

In contrast to the KIT repository, we inserted the link to our bug tracker directly into the resources:
https://github.com/hso-praktomat/praktomat/blame/master/README.md#L8

Reply:
:-) I got it :-) 
thanks for pointer.

Reply:
I removed this lines.